doc/source/serve/doc_code/object_detection.py

```diff
@@ -0,0 +1,45 @@
+import torch
+from PIL import Image
+import numpy as np
+from io import BytesIO
+from fastapi.responses import Response
+from ray import serve
+from fastapi import FastAPI
+
+app = FastAPI()
+
+
+@serve.deployment(num_replicas=1, route_prefix="/")
+@serve.ingress(app)
+class APIIngress:
+    def __init__(self, object_detection_handle) -> None:
+        self.handle = object_detection_handle
+
+    @app.get(
+        "/detect",
+        responses={200: {"content": {"image/jpeg": {}}}},
+        response_class=Response,
+    )
+    async def detect(self, image_url: str):
+        image_ref = await self.handle.detect.remote(image_url)
+        image = await image_ref
+        file_stream = BytesIO()
+        image.save(file_stream, "jpeg")
+        return Response(content=file_stream.getvalue(), media_type="image/jpeg")
+
+
+@serve.deployment(
+    ray_actor_options={"num_gpus": 1},
```

Comment:
Will this example run in CI? I'm not sure if there's a GPU available.

Reply:
Added GPU runner.